tests/unit/writers/qbwriter/test_qbwriter.py

```diff
@@ -119,7 +213,7 @@ def test_output_new_code_list_csws_urls_with_uri_style_without_file_extensions()
     qb_writer = QbWriter(cube)
     with TemporaryDirectory() as temp_dir:
         temp_dir = Path(temp_dir)
-        qb_writer._output_new_code_list_csvws(temp_dir)
+        qb_writer._output_new_dimension_code_list_csvws(temp_dir)
```

Comment:
Double check if we can consolidate the dimension_code_list and attribute_code_list functions.